### ISSUE TITLE
Dependency not installed with --alt

### ISSUE THREAD

**Issue by rwb27** [acornejo/homebrew-quartz#2]
Hello,
  On an entirely fresh homebrew install (I deleted /usr/local and reinstalled), on OS X 10.9 (13A603), if I do:

brew tap acornejo/homebrew-quartz
brew install inkscape-quartz

it fails trying to install the dependencies - the last brew to be installed is 
==> Installing inkscape-quartz dependency: gtk+-quartz
==> Downloading http://ftp.gnome.org/pub/gnome/sources/gtk+/2.24/gtk+-2.24.11.tar.xz
Already downloaded: /Library/Caches/Homebrew/gtk+-quartz-2.24.11.tar.xz
==> ./configure --prefix=/usr/local/Cellar/gtk+-quartz/2.24.11 --disable-glibtest --with-gdktarget=quartz --disable-introspection
installed software in a non-standard prefix.

Alternatively, you may set the environment variables BASE_DEPENDENCIES_CFLAGS
and BASE_DEPENDENCIES_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

If I then run "brew install inkscape-quartz" again, it gives the error 
Error: inkscape-quartz dependency acornejo/quartz/pango-quartz not installed with:
  --alt

I've tried googling it, and I'm stumped.  What is this --alt?  I tried brew install pango-quartz --alt and also editing the formula to include --alt in the configure arguments, but neither of these helped.

Any suggestions appreciated!

Richard

**Comment by acornejo** [comment]
Hi @rwb27 

I believe this is happening because brew has updated its conditional dependency format (i didn't notice because I haven't used this formula in a while).

I droped the alt dependency, please update the formulas and try it again.

**Comment by rwb27** [comment]
Hello,
  thanks for your impressively fast reply!  I've tried again, both from where I got to last time and from a clean install of homebrew.  In both cases, I now get to gtk+-quartz, which stops after the ./configure line, with the following message:
==> ./configure --prefix=/usr/local/Cellar/gtk+-quartz/2.24.11 --disable-glibtest --with-gdktarget=quartz --disable-introspection
installed software in a non-standard prefix.

Alternatively, you may set the environment variables BASE_DEPENDENCIES_CFLAGS
and BASE_DEPENDENCIES_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

READ THIS: https://github.com/Homebrew/homebrew/wiki/troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/acornejo/homebrew-quartz/issues

  It's not clear to me that this is actually an error, but brew stops at this point, so clearly something is not happy.  I've not investigated this, so perhaps after some googling I will be enlightened; I think the ball is very much still in my court.  However, I thought it might be worth posting the initial result just in case.

thanks again,

Richard

**Comment by acornejo** [comment]
Thanks Richard,

I'll try to see if I can reproduce that error on a clean install of homebrew. If you learn anything new let me know.

**Comment by rwb27** [comment]
Hello,
  I dug a little deeper (used the -v flag) and it seems to to be finding atk.  I have explicitly installed it, and have made sure to set PKG_CONFIG_PATH to use the one in /usr/local/lib.  However, the build still fails on atk.  That's somewhat confusing, because it's definitely there...
  Perhaps something in my .bash_profile is wrong?

$ env
TERM_PROGRAM=Apple_Terminal
SHELL=/bin/bash
TERM=xterm-color
CLICOLOR=1
TMPDIR=/var/folders/nz/6xk2v8d1749692czcbjkt8t40000gq/T/
Apple_PubSub_Socket_Render=/tmp/launch-g7atgW/Render
TERM_PROGRAM_VERSION=326
OLDPWD=/usr/Local
USER=rbowman
LD_LIBRARY_PATH=/usr/local/lib:/usr/lib
__CF_USER_TEXT_ENCODING=0x1F7:0:2
PATH=/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/opt/X11/bin:/opt/local/bin:/opt/local/sbin:/usr/local/mysql/bin:/Users/rbowman/bin:/opt/local/bin:/opt/local/sbin:/usr/local/mysql/bin:/Users/rbowman/bin
__CHECKFIX1436934=1
PWD=/usr
LANG=en_GB.UTF-8
SHLVL=1
HOME=/Users/rbowman
LOGNAME=rbowman
PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
DISPLAY=/tmp/launch-XsDDsx/org.macosforge.xquartz:0
_=/usr/bin/env

Richard

**Comment by rwb27** [comment]
Hello,
  A little more poking around (sorry, I'm rather new to homebrew) led me to remove the ":optional" from the dependency of gtk+-quartz on atk.  This seems to have sorted it out. (It looks like homebrew very helpfully changes the pkg_config_path to include the relevant packages even if they are keg-only, but in so doing it ignores whatever you set in the environment variable; this meant it didn't find atk even though it was present.)
  A similar problem happened in inkscape-quartz, I needed to add a dependency on intltool to get it to build correctly.

So, in summary, the two things I needed to do were:
gtk+-quartz now requires atk (i.e. it is not optional)
inkscape-quartz now requires intltool

It's still not building, but it's much closer now; it gets as far as compiling inkscape but fails.  I should probably open a new issue about that if it's not something silly I've done.

thanks!

Richard

**Comment by corysimmons** [comment]
Same problems. Extra steps like this kind of defeats Brew's purpose for me.

For others needing CLI Inkscape, you can just install Inkscape from their website and CLI should be available.
